projects/packages/jitm/src/class-jitm.php

```diff
@@ -115,7 +115,7 @@ public function jitm_enqueue_files() {
 
 		wp_enqueue_script(
 			'jetpack-jitm-new',
-			Assets::get_file_url_for_environment( '_inc/build/jetpack-jitm.min.js', '_inc/jetpack-jitm.js' ),
+			Assets::get_file_url_for_environment( '_inc/build/jetpack-jitm.min.js', '_inc/jetpack-jitm.js', __FILE__ ),
```

Comment:
This isn't working for me either. It tries to request `.../wp-content/plugins/jetpack-dev/vendor/automattic/jetpack-jitm/src/_inc/build/jetpack-jitm.min.js?ver=1.7.3`. I don't think `__FILE__` needs to be passed to `Assets::get_file_url_for_environment()` here.

```suggestion
			Assets::get_file_url_for_environment( '_inc/build/jetpack-jitm.min.js', '_inc/jetpack-jitm.js' ),
```